compiler/damlc/tests/daml-test-files/RequiresNonInterface.daml

```diff
@@ -0,0 +1,14 @@
+-- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+-- Test the serializability checker for Daml-LF.
+-- We use the template typeclass and instances directly as otherwise the error
+-- is caught prior due to missing Eq and Show instances.
+
+-- @ERROR range=14:13-14:21; cannot require 'Bool' because it is not an interface
```

Comment:
Same here

```suggestion
-- @SINCE-LF-FEATURE DAML_INTERFACE
-- @ERROR range=14:13-14:21; cannot require 'Bool' because it is not an interface
```